packages/ckeditor5-ui/src/toolbar/block/blocktoolbar.ts

```diff
@@ -111,6 +115,11 @@ export default class BlockToolbar extends Plugin {
 	 */
 	private _blockToolbarConfig: ReturnType<typeof normalizeToolbarConfig>;
 
+	/**
+	 * DOM Emitter.
+	 */
+	private _domEmitter: DomEmitter = new ( DomEmitterMixin() )();
```

Comment:
This is not used. Is this needed? Maybe we should use this emitter instead of `this.buttonView` as an emitter?